src/test/scala/firrtlTests/fixed/FixedTypeInferenceSpec.scala

```diff
@@ -36,7 +36,7 @@ class FixedTypeInferenceSpec extends FirrtlFlatSpec {
       """circuit Unit :
         |  module Unit :
         |    input a : Fixed<10><<2>>
-        |    input b : Fixed<10>
+        |    input b : Fixed<10><<0>>
```

Comment:
Does doing this mean there is no longer a test with Fixed having a width specified without BP? Seems like an important test case

Reply:
added test case